### ISSUE TITLE
[qt5cpp] Segfault when accessing/deleting object after calling API containing an object as body param

### ISSUE THREAD

**Issue by etherealjoy** [swagger-api/swagger-codegen#7250]
##### Description

The generated API passed the object param in the request body as a copy. When the API returns the copy is deleted and the pointer references inside also get deleted. After this attempting to delete or access the object causes segfault since the pointers inside the object have been deleted when the copy is destroyed on API return.
e.g.
```C++
void configCreate(DockerConfigSpec body);
```
instead of 
```C++
void configCreate(DockerConfigSpec& body);
```
##### Swagger-codegen version

2.3.0
3.0.0
##### Swagger declaration file content or url

Any swagger file containing Objects with dynamically allocated members as part of the request body.

##### Suggest a fix/enhancement

Passing the body Param by reference solves the problem.

**Comment by etherealjoy** [comment]
@wing328 I have a fix for this in my fork, but I will create PR after the pending ones are integrated.

**Comment by etherealjoy** [comment]
Solved with #7340
